projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -4118,16 +4147,11 @@ private void serializeHostConfigs(Path testRigPath, Path outputPath,
                   hostConfig.getIptablesFile());
             String relativePathStr = _testrigSettings.getBasePath()
                   .relativize(path).toString();
-            if (!iptablesConfigurations.containsKey(relativePathStr)) {
-               for (String key : iptablesConfigurations.keySet()) {
-                  _logger.errorf("key : %s\n", key);
-               }
-               throw new BatfishException(
-                     "Key not found for iptables: " + relativePathStr);
+            if (iptablesConfigurations.containsKey(relativePathStr)) {
```

Comment:
I think this is a good question. I'm not sure I fully understand the parsing philosophy. @arifogel may have thoughts.